documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1910,6 +1910,33 @@ This also enables tracking DDL changes against captured tables, so if the schema
 This allows LogMiner to mine substantially faster but at the expense that DDL changes cannot be tracked.
 If the captured table(s) schema changes infrequently or never, this is the ideal choice.
 
+|[[oracle-property-log-mining-buffer-type]]<<oracle-property-log-mining-buffer-type, `+log.mining.buffer.type+`>>
+|`memory`
+|The buffer type controls how the connector will manage buffering transaction data. +
+ +
+`memory` - Uses the JVM process' heap to buffer all transaction data.
+This is the ideal choice if you don't expect a lot of long-running or large transactions. +
+The buffer state is not persisted across restarts and will need to be recreated from the current offset's SCN value. +
+ +
+`infinispan` - This uses an embedded Infinispan cache to buffer all transaction data and persist it to disk.
+This is the ideal choice if you expect long-running or large transactions.
+Since the buffer state is persisted across restarts, the buffer does not need to be recreated at restart.
+Using this mode requires the use of `log.mining.buffer.location` to designate where the cache files are to be written.
+
+|[[oracle-property-log-mining-buffer-location]]<<oracle-property-log-mining-buffer-location, `+log.mining.buffer.location+`>>
+|
+|Specifies a directory location to where the connector can read and write the buffer cache to disk.
+This directory should be shared and available across all nodes in a Kafka Cluster and should be unique for each connector deployment.
+
+|[[oracle-property-log-mining-buffer-drop-on-stop]]<<oracle-property-log-mining-buffer-drop-on-stop, `+log.mining.buffer.drop.on.stop+`>>
+|`false`
+|Whether or not to delete the buffer state when the connector stops in a graceful, expected way. +
+ +
+This setting only impacts buffer implementations that persist state across restarts, such as `infinispan`. +
+The default behavior is that the buffer state is always retained between restarts. +
+ +
+Set to `true` in only testing or development environments.
```

Comment:
Set to `true` only in testing or development environments.

Reply:
Nitpick, but in this case, _only_ modifies the entire phrase, rather than the adjectives that modify _environments_.